2021-08-18 10:43:56,232 CRITICAL config_server_log.py Критическая ошибка
2021-08-18 10:43:56,232 ERROR config_server_log.py Ошибка
2021-08-18 10:43:56,232 DEBUG config_server_log.py Отладочная информация
2021-08-18 10:43:56,232 INFO config_server_log.py Информационное сообщение
2021-08-18 10:45:18,680 CRITICAL config_server_log.py Критическая ошибка
2021-08-18 10:45:18,680 ERROR config_server_log.py Ошибка
2021-08-18 10:45:18,680 DEBUG config_server_log.py Отладочная информация
2021-08-18 10:45:18,680 INFO config_server_log.py Информационное сообщение
2021-08-18 10:47:07,770 CRITICAL config_server_log.py Критическая ошибка
2021-08-18 10:47:07,772 ERROR config_server_log.py Ошибка
2021-08-18 10:47:07,772 DEBUG config_server_log.py Отладочная информация
2021-08-18 10:47:07,773 INFO config_server_log.py Информационное сообщение
2021-08-18 10:53:47,186 CRITICAL config_server_log.py Критическая ошибка
2021-08-18 10:53:47,189 ERROR config_server_log.py Ошибка
2021-08-18 10:53:47,189 DEBUG config_server_log.py Отладочная информация
2021-08-18 10:53:47,189 INFO config_server_log.py Информационное сообщение
2021-08-18 11:03:15,474 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-18 11:03:20,517 INFO server.py Установлено соедение с ПК ('127.0.0.1', 52144)
2021-08-18 11:03:20,517 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1629273800.5177526, 'user': {'account_name': 'Guest'}}
2021-08-18 11:03:20,517 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1629273800.5177526, 'user': {'account_name': 'Guest'}}
2021-08-18 11:03:20,517 INFO server.py Cформирован ответ клиенту {'response': 200}
2021-08-18 11:03:20,517 DEBUG server.py Соединение с клиентом ('127.0.0.1', 52144) закрывается.
2021-08-18 11:03:45,465 CRITICAL config_server_log.py Критическая ошибка
2021-08-18 11:03:45,465 ERROR config_server_log.py Ошибка
2021-08-18 11:03:45,465 DEBUG config_server_log.py Отладочная информация
2021-08-18 11:03:45,465 INFO config_server_log.py Информационное сообщение
